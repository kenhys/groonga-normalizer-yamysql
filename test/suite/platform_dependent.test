register normalizers/yamysql

normalize NormalizerYaMySQLUnicodeCI "−—〜‖"
