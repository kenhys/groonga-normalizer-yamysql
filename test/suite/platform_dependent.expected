register normalizers/yamysql
[[0,0.0,0.0],true]
normalize NormalizerYaMySQLUnicodeCI "−—〜‖"
[[0,0.0,0.0],{"normalized":"ーー~|","types":[],"checks":[]}]
